packages/pilot/src/containers/Reprocess/Form/index.js

```diff
@@ -90,8 +90,7 @@ const ReprocessForm = ({
   holderName,
   loading,
   lockReason,
-  onReprocessWithAntifraud,
-  onReprocessWithoutAntifraud,
+  onReprocess,
```

Comment:
Aqui poderiam ser recebidas as referencias prontas desta função onReprocess para evitar aquele `() => onReprocess(true)` maroto.
```suggestion
 onReprocessWithoutAntifraud,
 onReprocessWithAntifraud,
```
Usando isso na chamada deste componente teríamos algo assim
```
onReprocessWithoutAntifraud={onReprocess(true)}
 onReprocessWithAntifraud=onReprocess(false)
```

Reply:
Feito!